src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -2630,5 +2630,187 @@ static void Main()
                 Diagnostic(ErrorCode.ERR_BadSKknown, "System").WithArguments("System", "namespace", "type").WithLocation(6, 17)
                 );
         }
+
+        [Fact]
+        public void DefaultNullableParameter()
+        {
+            var text = @"
+class C
+{
+    static void Main() { A(); B(); }
+
+    static void A(int? x = default) => System.Console.Write(x.HasValue);
+    static void B(int? x = default(int?)) => System.Console.Write(x.HasValue);
+}";
+            var comp = CreateStandardCompilation(text, parseOptions: TestOptions.Regular7_1, options: TestOptions.DebugExe);
+            comp.VerifyDiagnostics();
+            CompileAndVerify(comp, expectedOutput: "FalseFalse");
+
+            var tree = comp.SyntaxTrees.First();
+            var model = comp.GetSemanticModel(tree);
+
+            var def = tree.GetCompilationUnitRoot().DescendantNodes().OfType<LiteralExpressionSyntax>().Single();
```

Comment:
>var def = tree.GetCompilationUnitRoot().DescendantNodes().OfType<LiteralExpressionSyntax>().Single(); [](start = 12, length = 101)

Please also assert the same semantic information for the ```default(int?)``` expression in the signature of ```B```.